main.tf

```diff
@@ -91,25 +90,28 @@ resource "aws_security_group" "instance" {
 resource "aws_instance" "example" {
   ami                    = "ami-0889a44b331db0194"
   instance_type          = "t2.micro"
+  # List of security group IDs to associate with. 
   vpc_security_group_ids = [aws_security_group.instance.id]
-
+  # User data to provide when launching the instance. 
+  # Updates to this filed will trigger a start/stop of 
```

Comment:
**Field** is misspelled. Also, the `user data` itself does not trigger a restart so remove this comment and move it to the appropriate section in the file.